pages/confirmEmail.js

```diff
@@ -0,0 +1,33 @@
+import Head from 'next/head';
+import confirmEmailStyles from '@app/styles/confirmEmail.module.css';
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+import { Icon } from 'semantic-ui-react';
+import Link from 'next/link';
+
+export default function confirmEmail() {
+  return (
+    <>
+      <Head>
+        <title>Confirm Email - ConnectOrlando</title>
+        <meta name="description" content="Email Confirmation" />
+      </Head>
+      <main className={confirmEmailStyles.center}>
+        <Icon name="check circle" size="massive" />
```

Comment:
Nice! Could you add an updated screenshot of the page so we can take a look at the new icon?

Reply:
Updated!

Reply:
I am having trouble logging in, says I dont have access. I probably should be using the connectorlando email right?